fixtures/HttpKernel/DummyKernel.php

```diff
@@ -42,15 +45,15 @@ public function unserialize($serialized)
     /**
      * {@inheritdoc}
      */
-    public function handle(Request $request, $type = self::MASTER_REQUEST, $catch = true)
```

Comment:
This constant isn't available before Symfony 5.3. Either bump the minimum version or hardcode the int `1`.

Reply:
Will bump it in another PR